cognite_toolkit/_cdf_tk/commands/_profile.py

```diff
@@ -185,6 +187,251 @@ def _as_cell(self, value: PendingCellValue) -> str | Spinner:
         return str(value)
 
 
+class ProfileAssetCommand(ProfileCommand):
+    def __init__(self, print_warning: bool = True, skip_tracking: bool = False, silent: bool = False) -> None:
+        super().__init__(print_warning, skip_tracking, silent)
+        self.table_title = "Asset Profile for Hierarchy"
+        self.hierarchy: str | None = None
+        self.aggregators: dict[str, MetadataAggregator] = {}
+
+    class Columns:
+        Resource = "Resource"
+        Count = "Count"
+        DataSets = "DataSet"
+        DataSetCount = "DataSet Count"
+        Transformations = "Transformation"
+        RawTable = "Raw Table"
+        RowCount = "Rows"
+        ColumnCount = "Columns"
+
+    is_dynamic_table = True
+    _index_split = "-"
+    profile_row_limit = 10_000  # The number of rows to profile to get the number of columns.
+    # The actual limit is 1 million, we typically run this against 30 tables and that high limit
+    # will cause 504 errors.
+    profile_timeout_seconds = 60 * 4  # Timeout for the profiling operation in seconds,
```

Comment:
Any particular reason why `60 * 4` ?

Reply:
I had a discussion with the team if I am remembering correctly.